src/adv_diff/BrinkmanAdvDiffSemiImplicitHierarchyIntegrator.cpp

```diff
@@ -856,6 +868,28 @@ BrinkmanAdvDiffSemiImplicitHierarchyIntegrator::registerBrinkmanAdvDiffBcHelper(
     return;
 } // registerBrinkmanAdvDiffBcHelper
 
+void
+BrinkmanAdvDiffSemiImplicitHierarchyIntegrator::setBrinkmanTimeIndepedent(Pointer<CellVariable<NDIM, double> > Q_var,
+                                                                          bool brinkman_time_independent)
+{
+    d_brinkman_time_independent[Q_var] = brinkman_time_independent;
+    return;
+} // setBrinkmanTimeIndepedent
+
+void
+BrinkmanAdvDiffSemiImplicitHierarchyIntegrator::registerTransportedQuantity(Pointer<CellVariable<NDIM, double> > Q_var,
+                                                                            bool output_Q)
+{
+#if !defined(NDEBUG)
+    TBOX_ASSERT(Q_var);
+    TBOX_ASSERT(std::find(d_Q_var.begin(), d_Q_var.end(), Q_var) == d_Q_var.end());
+#endif
+    AdvDiffSemiImplicitHierarchyIntegrator::registerTransportedQuantity(Q_var, output_Q);
+    d_brinkman_time_independent[Q_var] = false;
+    return;
```

Comment:
Let's use `d_Q_time_independent` instead of  `d_brinkman_time_independent` as it is used for all Q vars.